### PR TITLE
dbus: rename from d-bus

### PR DESCRIPTION
**Pull request by vladshablinsky**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Fixes https://github.com/Homebrew/legacy-homebrew/pull/42943

### CODE REVIEW

File: formula_renames.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "crystal": "autocode",
   "cv": "progress",
   "cyassl": "wolfssl",
+  "d-bus":"dbus",
```
Comment (UniqMartin) [inline formula_renames.json]: A space is missing between `"d-bus":` and `"dbus",`.

**Comment by vladshablinsky** [comment]
Done. Btw `brew install emacs --with-d-bus` works fine.

**Comment by mikemcquaid** [comment]
@vladshablinsky Can you add `revision 1` to the things that depend on it: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1586/

**Comment by vladshablinsky** [comment]
@mikemcquaid To all of them, or only to those that don't have `:optional`?
I think I also need to change `d-bus` to `dbus`, so audit doesn't warn.

Here is a list:

```
../Taps/homebrew/homebrew-core/Aliases/qt4:  depends_on "d-bus" => :optional
../Taps/homebrew/homebrew-core/Formula/at-spi2-core.rb:  depends_on "d-bus"
../Taps/homebrew/homebrew-core/Formula/dbus-glib.rb:  depends_on "d-bus"
../Taps/homebrew/homebrew-core/Formula/efl.rb:  depends_on "d-bus"
../Taps/homebrew/homebrew-core/Formula/emacs.rb:  depends_on "d-bus" => :optional
../Taps/homebrew/homebrew-core/Formula/gconf.rb:  depends_on "d-bus"
../Taps/homebrew/homebrew-core/Formula/gimp.rb:  depends_on "d-bus"
../Taps/homebrew/homebrew-core/Formula/pulseaudio.rb:  depends_on "d-bus" => :optional
../Taps/homebrew/homebrew-core/Formula/qt.rb:  depends_on "d-bus" => :optional
../Taps/homebrew/homebrew-core/Formula/qt5.rb:  depends_on "d-bus" => :optional
../Taps/homebrew/homebrew-core/Formula/wireshark.rb:  depends_on "d-bus"
```

**Comment by DomT4** [comment]
Anything recommended or mandatory normally.

**Comment by mikemcquaid** [comment]
Yep 👍.

**Comment by vladshablinsky** [comment]
@mikemcquaid something like this?

**Comment by DomT4** [comment]
Could you possibly pull the `d-bus` dependency rename for `qt` and `qt5` out of this PR so I don't have to bump the timeout to north of 5 hours to get this all to pass? 😅. We should be fairly safe to not revision the `:optional` dependencies as well, FWIW.

The `revision`s inside `bottle do` blocks aren't designed to be modified by humans. We know it's confusing to use the same word (and have plans to do something about it eventually) but formula revisions and bottle revisions are separate concepts.

**Comment by UniqMartin** [comment]
> We should be fairly safe to not revision the `:optional` dependencies as well, FWIW.

We definitely don't need to bump the revision where DBus is `:optional`. We should be updating the formulae to use the new name, but this can happen later in a separate PR.

The first commit should continue to be the one renaming `d-bus` to `dbus`. It should be followed by a series of commits (with one commit per changed formula) and a subject line like `<formula>: revision for dbus rename` for all affected formulae with a mandatory dependency on DBus. Thanks! 😸

**Comment by vladshablinsky** [comment]
@UniqMartin @DomT4 Updated. :)

**Comment by DomT4** [comment]
`midgard2` needs a revision too then, I guess. I presume it links through `dbus-glib` so the dependency wasn't directly stated.

**Comment by DomT4** [comment]
Everything passed this time around related to this PR. There's a syntax issue with the `brew` repo at the moment I'm sitting on in case Andrew wants to fix it himself, but it doesn't impact the actual build process, etc here.

Merged. Thank you once again @vladshablinsky, this one took a bit of work to get done 😅.

If you want to submit a PR to change the other `d-bus` dependencies that are only optional I'll merge that without needing to wait for CI, but entirely up to you whether you wish to do so or not.

**Pull request closed without merging**